src/.nuget/toolchain.ubuntu.14.04-arm.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>toolchain.ubuntu.14.04-arm.Microsoft.DotNet.ILCompiler.Development</id>
```

Comment:
@manu-silicon In this case, ID should follow the same name as the nuget package.